### ISSUE TITLE
Add School Districts?

### ISSUE THREAD

**Issue by kayleachampion** [phildini/stayinghomeclub#1287]
I see there's a category for Universities, would it also make sense to add School Districts?

**Comment by virtadpt** [comment]
Yes, please.  Especially because some (like mine) have been waffling about it since the beginning.

**Comment by drewbrew** [comment]
I think adding school districts is way too much for us to track at this point. You're welcome to put something together, but we don't have the time to do so.
